### PR TITLE
[6.12.z] [virt-who-config]-upgrade scenario refactor

### PR DESCRIPTION
**Pull request by Satellite-QE**
Cherrypick of PR: https://github.com/SatelliteQE/robottelo/pull/11107

Test PASS:
```
================================================================================== 1 passed, 3 deselected, 10 warnings in 10.52s ==================================================================================
(robottelo_vv) [virtwho@dell-per740-68-vm-04 robottelo]$ pytest ./tests/upgrades/test_virtwho.py  -m pre_upgrade --disable-pytest-warnings
=============================================================================================== test session starts ===============================================================================================
platform linux -- Python 3.10.9, pytest-7.2.2, pluggy-1.0.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/virtwho/gitrepo/repo/robottelo, configfile: pyproject.toml
plugins: services-2.2.1, mock-3.10.0, reportportal-5.1.5, ibutsu-2.2.4, cov-3.0.0, xdist-3.2.1
collected 2 items / 3 deselected / -1 selected                                                                                                                                                                    

tests/upgrades/test_virtwho.py .                                                                                                                                                                            [100%]

============================================================================ 1 passed, 3 deselected, 19 warnings in 119.54s (0:01:59) =============================================================================
```
```
(robottelo_vv) [virtwho@dell-per740-68-vm-04 robottelo]$ pytest ./tests/upgrades/test_virtwho.py  -m post_upgrade  --disable-pytest-warnings
=============================================================================================== test session starts ===============================================================================================
platform linux -- Python 3.10.9, pytest-7.2.2, pluggy-1.0.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/virtwho/gitrepo/repo/robottelo, configfile: pyproject.toml
plugins: services-2.2.1, mock-3.10.0, reportportal-5.1.5, ibutsu-2.2.4, cov-3.0.0, xdist-3.2.1
collected 2 items / 3 deselected / -1 selected                                                                                                                                                                    

tests/upgrades/test_virtwho.py .                                                                                                                                                                            [100%]

================================================================================== 1 passed, 3 deselected, 10 warnings in 10.59s ==================================================================================

```

### CODE REVIEW

**Review by pondrejk — APPROVED** [review]

File: tests/upgrades/test_virtwho.py [hunk 1]
```diff
@@ -84,18 +83,19 @@ def test_pre_create_virt_who_configuration(
         )
         default_loc = target_sat.api.Location(id=default_loc_id).read()
         org = target_sat.api.Organization(name=ORG_DATA['name']).create()
-        default_loc.organization.append(entities.Organization(id=org.id))
+        default_loc.organization.append(target_sat.api.Organization(id=org.id))
         default_loc.update(['organization'])
+        org.sca_disable()
```
Comment (Gauravtalreja1) [inline tests/upgrades/test_virtwho.py:88]: In 6.12 or below versions SCA is disabled already, so I think it is not required, @yanpliu can you please check and update?

**Review by Gauravtalreja1 — APPROVED** [review]
Ack, with a question

File: tests/upgrades/test_virtwho.py (hunk 1 above)
Comment (yanpliu) [reply tests/upgrades/test_virtwho.py:88]: @Gauravtalreja1 Yes , you are right, this update is not for satellite6.12 , it is for satellite6.13 . Satellite6.12 is SCA disabled by default

**Review by shubhamsg199 — APPROVED** [review]
ACK

**Review by JacobCallahan — APPROVED** [review]

**Pull request merged**